src/theme.c

```diff
@@ -25,7 +25,11 @@
 #include "common/string-helpers.h"
 #include "config/rcxml.h"
 #include "button/button-png.h"
+
```

Comment:
We need here `#include "config.h"`

Reply:
..or probably the line above.

Reply:
..maybe it should go in `include/theme.h` ? Perhaps future proofing?

Reply:
Fixed. Thanks for finding that.